client/src/components/Header/HeaderSearch.tsx

```diff
@@ -1,7 +1,39 @@
 import React from 'react';
+import styled from '@emotion/styled';
+
+import { Search } from '@/assets/svgIcons';
+
+const SearchWarepper = styled.div`
+  place-self: center end;
+  display: flex;
+  align-items: center;
+`;
+
+const SearchInput = styled.input`
+  padding: 0.4rem 0;
+  width: 15rem;
+  font-size: 0.85rem;
+  border-style: none;
+  border-bottom: 1px solid gray;
+  outline: none;
+`;
+
+const SearchButton = styled.button`
+  background-color: rgba(0, 0, 0, 0);
+  border-style: none;
+  width: 2.2rem;
+  cursor: pointer;
+`;
 
 const HeaderSearch = (): JSX.Element => {
-  return <div>HeaderSearch</div>;
+  return (
+    <SearchWarepper>
+      <SearchInput placeholder="검색어를 입력해주세요"></SearchInput>
```

Comment:
```suggestion
      <SearchInput placeholder="검색어를 입력해주세요" />
```

내용이 빈 경우엔 하나로 쓰면 좋을 것 같군요 ㅎㅎ
input 내부엔 아무것도 넣을 수 없기도 하구요

Reply:
수정했씁니다!